packages/builder/src/pages/builder/app/[application]/data/datasource/[selectedDatasource]/index.svelte

```diff
@@ -92,28 +171,62 @@
         <Divider />
         <div class="query-header">
           <Heading size="S">Tables</Heading>
-          <Button primary on:click={updateDatasourceSchema}
-            >Fetch Tables From Database</Button
-          >
+          <div class="table-buttons">
+            {#if plusTables && plusTables.length !== 0}
+              <Button primary on:click={openDisplayColumnModal}>
+                Update display columns
+              </Button>
+            {/if}
+            <div>
+              <Button primary on:click={updateDatasourceSchema}>
+                Fetch Tables From Database
```

Comment:
Super insignificant (and there are still plenty of place in the builder that need updated) but Adobe's recommendation is always to use normal sentence case for buttons - so just capitalise the first letter of the first word and lower case the rest. So just "Fetch tables from database" here, like it is with the display columns button.